ros_ign_bridge/package.xml

```diff
@@ -24,16 +24,16 @@
   <depend>trajectory_msgs</depend>
 
   <!-- Garden -->
-  <depend condition="$IGNITION_VERSION == garden">gz-msgs9</depend>
-  <depend condition="$IGNITION_VERSION == garden">gz-transport12</depend>
+  <depend condition="$GZ_VERSION == garden or $IGNITION_VERSION == garden">gz-msgs9</depend>
```

Comment:
Just a heads up that the syntax for these conditions can be finicky at times. It needs to be understood by `colcon`, `rosdep` and `bloom`. I've checked that `colcon` and `rosdep` work, but I think we'll only know for sure with `bloom` once we make a release